composer.json

```diff
@@ -38,7 +39,10 @@
     "extra": {
         "branch-alias": {
             "dev-master": "4.0.x-dev"
-        }
+        },
+        "expose": [
+            "client"
```

Comment:
I feel this should be a bit more restricted to `client/dist` or even `client/dist/js` and `client/dist/styles`

Reply:
Good call - updated